### ISSUE TITLE
A mechanism for users to block abusive users

### ISSUE THREAD

**Issue by codesworth** [entaq/floq#40]
This is counter productive as our users do not interact directly. We may have to appeal this to the Appstore team. Or do u have any ideas about this

**Comment by jberkey78** [comment]
I have plans to add the ability to follow users and accounts, so this will be relevant when that time comes. Right now the flagging of content acts in a similar way. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by jberkey78** [comment]
below the create by info lets add a line of copy.

You can block users and their content by swiping left on their user name, this action can be undone.

**Comment by jberkey78** [comment]
![IMG_83A678FE31CD-1](https://user-images.githubusercontent.com/13186530/57587831-9b13c480-74d9-11e9-87e8-bba17857eb98.jpeg)

**Comment by jberkey78** [comment]
Pictured above.

I am looking at a Cliq where I have been blocked by the user BeccaBerkey, it shows I am blocked in my view.  It should just not show the user who blocked me and hide their photos so I don't know they are there.

This will need fixed before resubmission.

**Comment by codesworth** [comment]
Alright, will fix this in next update. Are we to implement a mechanism for unblocking users?

**Comment by jberkey78** [comment]
They didn’t say we needed too, but the functionality is already there. I think it’s a good thing to have as long as it doesn’t create other issues. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by codesworth** [comment]
Will add an another option in user setting that lists blocked users to allow users to unbloked them. We will check it in the next update. if not necessary we can take it out.

**Comment by jberkey78** [comment]
So currently if I block a user I can unblock them, that already works. But the user who has been blocked should not see the user who blocked them, even if they are in the same cliq 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by jberkey78** [comment]
I’ll try to do a screen recording of the current interaction 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by jberkey78** [comment]
This is my account and Becca blocked me, it shows me being blocked and her in cliq. Her photos are hidden. She should just not show up since she blocked me and it should not say someone blocked me, that should only show show for her. 

![GitHawk Upload by jberkey78](https://i.imgur.com/NW33cDE.jpg)

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by codesworth** [comment]
Yeah I was contemplating on that too when i wrote that feature, I will make the changes.
